package.json

```diff
@@ -11,7 +11,7 @@
   "scripts": {
     "start": "node index.js",
     "dev": "./node_modules/.bin/nodemon index.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "mocha"
```

Comment:
Did you consider using Jest?

Reply:
Yes, Jest was the first library that I tried. But I had issues getting started with it. [There was some error about the module syntax not being correct.](https://stackoverflow.com/questions/61781271/jest-wont-transform-the-module-syntaxerror-cannot-use-import-statement-outsi) After trying a few things and reading about other testing libraries being less painful to bootstrap, I decided to use Mocha.
By the way, all these module import errors are quite confusing, at least to me (ES vs common JS modules). :confused: